dashing/distribution.yaml

```diff
@@ -1917,6 +1917,22 @@ repositories:
       url: https://github.com/ros-visualization/qt_gui_core.git
       version: crystal-devel
     status: maintained
+  rc_dynamics_api:
+    doc:
+      type: git
+      url: https://github.com/roboception/rc_dynamics_api.git
+      version: master
```

Comment:
I see that both the ROS 1 and ROS 2 versions of this package point to the master branch of the repository.  Does it work for both of them simultaneously?

Reply:
yes, as that is a non-ROS third party package that is then used in separate ROS1 and ROS2 packages as system dependency...

Reply:
Sounds good, just checking.

Reply:
Thanks for staying vigilant :-)